packages/amplication-client/src/Project/ProjectSideBar.tsx

```diff
@@ -1,37 +1,45 @@
-import React from "react";
+import React, { useCallback } from "react";
+import { useHistory } from "react-router-dom";
 import { Project, Workspace } from "../models";
 import WorkspaceSelector from "../Workspaces/WorkspaceSelector";
+import AddNewProject from "./AddNewProject";
 import ProjectList from "./ProjectList";
 import "./ProjectSideBar.scss";
 import ProjectSideBarFooter from "./WorkspaceSettingsNavigation";
 
 const CLASS_NAME = "project-sidebar";
 
 type Props = {
-  currentWorkspace: Workspace;
-  projectsList: Project[];
-  setNewProject: (project: Project) => void;
+  currentWorkspace: Workspace | undefined;
+  projectsList: Project[] | null;
 };
 
-const ProjectSideBar = ({
-  currentWorkspace,
-  projectsList,
-  setNewProject,
-}: Props) => {
-  return (
+const ProjectSideBar = ({ currentWorkspace, projectsList }: Props) => {
+  const history = useHistory();
+
+  const handleProjectCreated = useCallback(
+    (project: Project) => {
+      history.push(`/${currentWorkspace?.id}/${project.id}`);
+    },
+    [currentWorkspace, history]
+  );
+
+  return currentWorkspace ? (
     <div className={CLASS_NAME}>
       <span className={`${CLASS_NAME}__label`}>Workspace</span>
       <WorkspaceSelector />
       <hr className={`${CLASS_NAME}__divider`} />
-      <ProjectList
-        projects={projectsList}
-        handleProjectChange={setNewProject}
-        workspaceId={currentWorkspace.id}
-      />
+      {ProjectList && (
```

Comment:
@abrl91 Did you mean to check `projectsList` :)

Reply:
Not really, it was just for a test and it was left by mistake. Thank you